src/components/code-line.component.js

```diff
@@ -130,6 +130,7 @@ export class CodeLine extends Component {
                 CodeTag={Text}
                 codeTagProps={{ style: styles.codeLine }}
                 customStyle={customStyle}
+                fontFamily={fonts.fontCode.fontFamily}
```

Comment:
Any chance we can have `...fonts.fontCode` spread within `customStyle` instead of being applied as a separate prop?

If `fontFamily` has to be applied for some reason, than that's not a problem